themes/LaserWave-color-theme.json

```diff
@@ -66,6 +66,20 @@
                 "foreground": "#EB64B9"
             }
         },
+        {
+			"name": "Html punctuations tags",
+			"scope": "punctuation.definition.tag.end, punctuation.definition.tag.begin, punctuation.definition.tag",
```

Comment:
All punctuation should be #716385 to keep in line with the rest of the theme so if you could move these scopes into the punctuation section that would be great.

With the below tags the color is correct but I'm trying to keep the project with few tokenColors to keep it easy to maintain, so if you could move those into the already defined "Tags" tokenColor I will def merge this.

Thanks for contributing!